test/tests/functional/pbs_resc_used_single_node.py

```diff
@@ -147,7 +147,7 @@ def test_epilogue_single_node(self):
 """
 
         hook_name = "epi"
-        a = {'event': "execjob_epilogue", 'enabled': 'True'}
+        a = {'event': "execjob_epilogue", 'enabled': 'True', 'order': '200'}
```

Comment:
Add a comment explaining why you need order=200.  If the cgroups order is ever changed, and this test starts failing again, it will be helpful to know what to do.